workflow/applications/gfs_forecast_only.py

```diff
@@ -129,6 +129,23 @@ def get_task_names(self):
             if self.do_awips:
                 tasks += ['waveawipsbulls', 'waveawipsgridded']
 
+        if self.do_bufrsnd:
+            tasks += ['postsnd']
+
+        if self.do_gempak:
+            tasks += ['gempak']
+            tasks += ['gempakmeta']
+            tasks += ['gempakncdcupapgif']
+            tasks += ['gempakpgrb2spec']
+
+        if self.do_awips:
+            tasks += ['awips_20km_1p0deg']
+            tasks += ['awips_g2']
+            tasks += ['fbwinds']
+
+        if self.do_wafs:
+            tasks += ['wafs', 'wafsgcip', 'wafsgrib2', 'wafsgrib20p25', 'wafsblending', 'wafsblending0p25']
```

Comment:
This was moved earlier in the script in #2110 (except WAFS, which was deleted). Please make any changes in the new section and remove this.